### ISSUE TITLE
Account profile pages should have a clear and transparent grade context displayed + algorithm for selecting default context 

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#2526]
On the profile page there is some opaque logic which decides what context the profile should be display in. 

I want to make it clearer how this is chosen, find a better way to choose it, and expose this in the UI.

How to detect the grade auto-context:

* [ ] Initially it will use the persons country when their are no ticks. We still want a context for everyone even if they haven't ticked yet.

* [ ] After they start ticking we tally up all the systems for each tick, then find the context which covers the most of those systems. See the comment below for an example of why

* [x] display this context in the header exactly the same as the context we show at the node level

![image](https://cloud.githubusercontent.com/assets/187449/21959203/09ae4602-db14-11e6-9441-5e8033fb374b.png)

* [ ] where grades in the various tables differ from this context then we should the grade helper the same as we do in the index view, eg "AU: 25" if my context was US.

**Comment by brendanheywood** [comment]
A new mockup showing the dual lang + grades display and choice:

![image](https://cloud.githubusercontent.com/assets/187449/21959780/622820f2-db26-11e6-8830-1c047d1ec6b6.png)

**Comment by brendanheywood** [comment]
I've just reworked this algorithm todo list in the issue desc to better cover some edge cases. Some justification for this is below:

1) Imagine I'm a new account, I'm a yank with my country set to USA. Clearly my auto-context should be US.

2) I go bouldering a bit, I have 20 ticks for V-grade in the US context. Clearly my auto-context should still be US.

3) I dabble in trad and do 5 YDS routes in the US context, Clearly my auto-context should still be US.

4) I move to Aus, I change my country to Aus, but nothing else has changed, my auto-context should still be US.

5) I do some more bouldering in Aus, 10 boulders using V-grades in AU context, there is still no reason to change my context it should be US.

6) I do a mere 6 trad routes in Ewbank in the AU context. This I think warrants a swap of the auto-context to AUS.

It's not just about either the system or the context, it's about adding up all the systems and finding the context which covers as many of the routes as possible. If we choose US then the 6 Ewbank routes will show as 'AU:22', if we choose AU then only 5 US routes will show as 'US:5.11c'. If we did this based on systems alone we would choose US by a long way and I think be worse off for it.
